lms/djangoapps/grades/api/v1/views.py

```diff
@@ -754,17 +762,28 @@ def _create_override(self, subsection_grade_model, **override_data):
             grade=subsection_grade_model,
             defaults=self._clean_override_data(override_data),
         )
-        signals.SUBSECTION_OVERRIDE_CHANGED.send(
-            sender=None,
-            user_id=subsection_grade_model.user_id,
-            course_id=text_type(subsection_grade_model.course_id),
-            usage_id=text_type(subsection_grade_model.usage_key),
-            only_if_higher=False,
-            modified=override.modified,
-            score_deleted=False,
-            score_db_table=ScoreDatabaseTableEnum.overrides,
-            force_update_subsections=True,
+
+        set_event_transaction_type(SUBSECTION_GRADE_CALCULATED)
+        create_new_event_transaction_id()
+
+        apply_result = recalculate_subsection_grade_v3.apply(
+            kwargs=dict(
+                user_id=subsection_grade_model.user_id,
+                anonymous_user_id=None,
+                course_id=text_type(subsection_grade_model.course_id),
+                usage_id=text_type(subsection_grade_model.usage_key),
+                only_if_higher=False,
+                expected_modified_time=to_timestamp(override.modified),
+                score_deleted=False,
+                event_transaction_id=unicode(get_event_transaction_id()),
+                event_transaction_type=unicode(get_event_transaction_type()),
+                score_db_table=ScoreDatabaseTableEnum.overrides,
+                force_update_subsections=True,
+            ),
+            countdown=RECALCULATE_GRADE_DELAY_SECONDS,
```

Comment:
I don't think you want this - it will now sleep for 2 seconds before starting the task.